README.md

```diff
@@ -1 +1,59 @@
-# terraform-aws-account
\ No newline at end of file
+# terraform-aws-account
```

Comment:
please have some minimal description and an basic example module block here with terraform registry pointing source

have versions.tf with terraform and required providers versions/blocks